packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -26,9 +26,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     setupFiles: [resolve('config/polyfills.js')],
     setupTestFrameworkScriptFile: setupTestsFile,
-    testMatch: [
-      '<rootDir>/src/**/__tests__/**/*.js?(x)',
-      '<rootDir>/src/**/?(*.)(spec|test).js?(x)',
+    modulePathIgnorePatterns: [
+      '<rootDir>[/\\\\](build|docs|node_modules|scripts)[/\\\\]',
```

Comment:
I've added dist as it's a common folder. 
`.gitignore` really focuses on something different even if 99% of the time it's the same. 

Its better to accidentally watch a folder that doesn't need to be watched than to accidentally not-watch a folder that should be watched.